### ISSUE TITLE
Spline options in grViz-plotting

### ISSUE THREAD

**Issue by statpunx** [adayim/consort#23]
Dear Alim,

while using the "plot(g, grViz = T)" option (with a fairly big diagram with 4 arms) the connections are partly displayed as curved lines. Is there a way to implement spline options or does this require building a GraphViz object from scratch?
See the attached picture for reference.

![Unbenannt](https://github.com/adayim/consort/assets/100941032/2004d89a-7035-4f58-8ff2-0b5cc1f35b41)

**Comment by adayim** [comment]
Hi,

I've tried to measure the text inside the box and calculated the `minlen`, but it's not ideal. I've updated the package with options to avoid overlap and make sure the lines are straight with `splines=ortho, overlap=prism` option. Thanks for bringing up the `spline` option.

Alternatively, you can use `cat(build_grviz(g), file = "consort.gv")` to save the `grViz` and tweak it. I would prefer to improve this package and avoid manual editing if possible.

Can you try out the development version on GitHub and let me know if it solved your problem? Thank you very much.

Best,
Alimu

**Comment by statpunx** [comment]
Hi Alimu,

thank you so much for the quick answer and help! I ran my code again and the changes you made definetly improved things, no more curved lines! 

![Screenshot 2024-05-29 084318](https://github.com/adayim/consort/assets/100941032/16d12840-825a-499b-b9cc-46d34d8b710b)

Is there an option to adjust the scaling in this new version, so that the arms are closer to each other? I tried to find it in the development versions code, but since I'm not that familiar with GraphViz and your functions, I couldn't find anything. I can provide you with sample data to replicate my problem if that makes it easier for you, just let me know :)

All the best,
Thomas
